config/condition_catalog_services.go

```diff
@@ -96,13 +96,15 @@ func (c *CatalogServicesConditionConfig) Merge(o ConditionConfig) ConditionConfi
 	return r2
 }
 
-// Finalize ensures there no nil pointers.
+// Finalize ensures there no nil pointers with the _exception_ of Regexp. There
```

Comment:
I didn't really like making this exception here but couldn't figure out another way. Ideas welcome!

Reply:
This exception strikes me as fine!